crates/stages/src/db.rs

```diff
@@ -164,6 +164,26 @@ where
         Ok((prev_body.start_tx_id + prev_body.tx_count, last_transition))
     }
 
+    /// Get the result vector with range [start_key:end_key).
+    /// Please note that start_key is inclusive and end_key is expclusive.
+    #[allow(clippy::type_complexity)]
+    pub(crate) fn get_range<T: Table>(
```

Comment:
I'm think about the name of this function. First, I started it with `walk_range()`. But I changed it as `get_range()` because it returns `Vector` but doesn't return `Iterator`. HDYT?